PermissionScope/PermissionScope.swift

```diff
@@ -93,6 +98,17 @@ public class PermissionScope: UIViewController, CLLocationManagerDelegate, UIGes
     var authChangeClosure: ((Bool, [PermissionResult]) -> Void)? = nil
     var cancelClosure: (([PermissionResult]) -> Void)? = nil
 
+    // Computed variables
+    var allAuthorized: Bool {
```

Comment:
Can you apply these computed variables in `show` and `detectAndCallback` where appropriate?